### PR TITLE
Feature/cg 1189 support az nist encryption in transit rules

### PR DESCRIPTION
**Pull request by zhouse51**

### CODE REVIEW

File: src/aws/cis-1.2.0/CHANGELOG.md
Comment (tyler-dunkel) [inline src/aws/cis-1.2.0/CHANGELOG.md:1]: not sure why this is in here, can you rebase on alpha?

File: src/aws/cis-1.2.0/package.json
Comment (tyler-dunkel) [inline src/aws/cis-1.2.0/package.json:4]: 🤔 hmm maybe the repo wasnt fast forwarded from main?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:142]: @aacotroneo is this a rule that will need to be rewritten once we have `auzureSubscription`?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:150]: are there any log alerts that arent global?

File: src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts
Comment (tyler-dunkel) [inline src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts:13]: are these exported types needed?

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts
Comment (aacotroneo) [reply src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:142]: yes

File: src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts
Comment (m-pizarro) [reply src/azure/nist-800-53-rev4/rules/azure-nist-800-53-rev4-3.7.ts:150]: Not really, we can simplify it and remove that condition

File: src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts
Comment (m-pizarro) [reply src/azure/nist-800-53-rev4/tests/nist-800-53-rev4-6.x.test.ts:13]: No, I will remove everything that is not used

**Review by tyler-dunkel — APPROVED** [review]

**Comment by tyler-dunkel** [comment]
@m-pizarro this should be good to go after merge conflict resolved

**Comment by m-pizarro** [comment]
> @m-pizarro this should be good to go after merge conflict resolved

@tyler-dunkel  done!

**Pull request merged**
